quickwit-indexing/src/source/mod.rs

```diff
@@ -161,9 +162,27 @@ pub fn quickwit_supported_sources() -> &'static SourceLoader {
     })
 }
 
-#[derive(Clone)]
+/// A `SourceConfig` describes the properties of a source. A source config can be created
+/// dynamically or loaded from a file consisting of a JSON object with 3 mandatory properties:
+/// - `source_id`, a name identifying the source uniquely;
+/// - `source_type`, the type of the target source, for instance, `kafka` or `kinesis`;
```

Comment:
do we have a working kinesis source?

Reply:
No, not yet. I'll remove the mention to Kinesis.